lib/matplotlib/patches.py

```diff
@@ -3102,113 +3102,43 @@ def transmute(self, path, mutation_size, linewidth):
     class Curve(_Curve):
         """A simple curve without any arrow head."""
 
-        def __init__(self):
-            super().__init__(beginarrow=False, endarrow=False)
+        # These attributes only matter if someone modifies beginarrow/etc. on
+        # an ArrowStyle instance.
+        head_length = .2
+        head_width = .1
```

Comment:
This doesn't get overwritten by the super `__init__`, which has different defaults?

Reply:
I think that's correct for backcompat?  Previously, `Curve.__init__` would call `_Curve.__init__` and use the default head_length and head_width from it, which were .2, .1 (and not .4, .2 as they are for all other `_Curve` subclasses), so I just hard-coded these values here?

Reply:
But the `Curve.__init__` call to `super().__init__()` passes nothing, so the defaults from the signature will be used, which will overwrite these two.

Reply:
ah, indeed; moved them to instance attributes instead.

Reply:
Or you could pass the different default to `super().__init__()`.

Reply:
Indeed, I guess that's a bit nicer.